force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +224,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
-
+        List<Integer> intList = new List<Integer>{1,2,3};
         // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
+        // add 3 
+        // add 4
+        // add 5
+        intList.add(3);
+        intList.add(4);
+        intList.add(5);
```

Comment:
good if you want you can use the List constructor to be more concise. 

 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }